sunpy/util/scraper.py

```diff
@@ -331,3 +332,42 @@ def _smallerPattern(self, directoryPattern):
                 return None
         except Exception:
             raise
+
+    def _extractMetaURLs(self, timerange, extractor=None, translator=None):
```

Comment:
This function name isn't great, python method names are supposed to use snake_case.

I am not sure what a good name for this function is, it's doing a few things (calling filelist as well as running things through parse). Do we want to see this become the main entry point for scraper in a land where you use parse for everything?

Reply:
Yes, `filelist` will become internal function for scraper and the client would only be using the `extractMetaURLs` method.